server/src/app.module.ts

```diff
@@ -10,7 +12,9 @@ import { ApolloDriver } from '@nestjs/apollo';
     GraphQLModule.forRoot({
       autoSchemaFile: 'schema.gql',
       driver: ApolloDriver
-    })
+    }),
+    MongooseModule.forRoot('mongodb://localhost:27017/nist'),
```

Comment:
Could we make this pull from config service?

Reply:
Yes it should be, that'll be added in soon